cmd/criticize/main.go

```diff
@@ -111,6 +118,25 @@ func parseArgv(l *linter) {
 	}
 }
 
+func (l *linter) LoadConfig() {
+	raw, err := ioutil.ReadFile(l.configFile)
+	if err != nil {
+		return
+	}
```

Comment:
This will make specifying non-existing file error impossible to detect.
Maybe we should treat default argument specifically, but if user specified different filename, then exit with error if that config not found?